CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Change Log
 
+Version 0.18.0 *(2021-09-13)*
+---------------------------------
+
+- **Behavior Change:** When using version 7.1.0 or newer of the Android Gradle Plugin we will now publish all variants
+of a library unless `androidVariantToPublish` was set in the DSL. This means that for example both `debug` and `release`
```

Comment:
Why are we telling them to use something that's deprecated?

Reply:
It's meant to say that if you already set it explicitly we are not automatically forcing the behavior change and that the property still works while it is still there